app/models/complete_moab.rb

```diff
@@ -26,14 +26,6 @@ class CompleteMoab < ApplicationRecord
   # NOTE: size here is approximate and not used for fixity checking
   validates :size, numericality: { only_integer: true, greater_than: 0 }, allow_nil: true
 
-  scope :by_moab_storage_root_name, lambda { |name|
-    joins(:moab_storage_root).where(moab_storage_roots: { name: name })
-  }
-
-  scope :by_storage_location, lambda { |storage_dir|
```

Comment:
wow do we not use this anywhere?

Reply:
Not after my revisions.  Basically, the use case for most of our "scopes that join" should be:
1. do a `.find_by!` on the thing you are join on
2. use the association to get objects of the type where the scope used to live (which can still be chained w/ other scopes)

This reads a lot better, and if it fails (the `find_by!`) it fails in a much more obvious way.  Less custom code for us to track.